### ISSUE TITLE
Make sure everything works after npm CouchDB upgrade

### ISSUE THREAD

**Issue by zarianec** [nice-registry/all-the-package-repos#32]
Npm is going to upgrade the CouchDB and reset the sequence number as a result. Need to make sure that synchronization will continue to work after that.

This may require a manual update of the metadata or even a full rebuild.

See: https://github.blog/changelog/2021-09-08-npm-couchdb-upgrade-will-reset-sequence-number/

**Comment by zeke** [comment]
> The npm registry will upgrade its public CouchDB instance on September 15, 2021. As part of this upgrade, we will promote a new CouchDB replica to become our new public CouchDB instance, https://replicate.npmjs.com. This upgrade will reset the CouchDB sequence number.

T minus 4️⃣  days...

**Comment by eridal** [comment]
Thanks for letting us know! 

This should not be complicated, it's just a matter of..

1. Stop github actions
2. Wait till sequence is updated..
3. Clone the repo, remove the metadata, re-run the updater locally
4. Commit the new metadata and packages, along with `npm version minor`
5. Start github actions

**Comment by zeke** [comment]
@eridal thanks for summarizing that. Would you be willing to do that process?

**Comment by eridal** [comment]
Yeah, I can take care. We could also use the opportunity to apply changes that require history reprocess, so two birds one stone.. but I don't want to rush those changes.

Also fun fact they did not specify the exact date in UTC so we should stop the actions today:

> The npm registry will upgrade its public CouchDB instance on September 15, 2021. As part of this upgrade, we will...

**Comment by eridal** [comment]
oh another fun fact in UTC it **already** is 15/11 :))

**Comment by eridal** [comment]
Actions disabled

**Comment by eridal** [comment]
Confirming npmjs finished the upgrade and sequence number changed. Later today I will leave the updater running, probably overnight.

**Comment by zeke** [comment]
🍿

**Comment by eridal** [comment]
updater finished running, I'm wrapping up and will be publishing a new minor version during the next few minutes.

Just for reference here are the steps needed to trigger a manual full reprocess:

```shell
# allow cache, useful to save time on re-executions
mkdir -p .cache
export CACHE_DIR=.cache
export CACHE_READ=1
export CACHE_WRITE=1

# disable the timeout
export KILL_AFTER_MILLIS=Infinity

# clean all and execute
rm data/*
npm run update
```

**Comment by eridal** [comment]
Update complete. Actions re-enabled but only every 12hs, just to make sure they work correctly.

**Comment by zeke** [comment]
Nice! Thanks @eridal for taking care of that and for documenting the process. ⭐
